src/studio/nuxeo-studio/working-in-studio/migration-services.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration Services
+review:
+    comment: ''
+    date: '2018-12-05'
+    status: ok
+tree_item_index: 950
+toc: true
+---
+The JSF UI to Web UI migration tool is a safe and automated way to create, within a few minutes, an equivalent Web UI configuration based on your existing JSF UI configuration.
+
+The migration tool reduces dramatically the effort needed to transition to [Web UI]({{page space="nxdoc" page="web-ui"}}), our user interface that uses modern web standard technologies to provide an outstanding user experience.
```

Comment:
I'm not keen about this ideal vision. We have to make clear the first iteration is for simple document type and that we will handle complex logic little by little.

I would remove the "our user interface that..." which doesn't bring much info